### ISSUE TITLE
 Setup a vertical offset for the headset marker

### ISSUE THREAD

**Issue by Djneo134** [NOLOVR/NOLO-Driver-For-Windows#17]
Hi,
Is there a way to apply a vertical offset to the Nolo headset marker ?
Every time, I need to put the headset on the floor to calibrate it. Which is not cool at all.

I would like to put it on my desk then to apply an offset of 80 cm vertically when I click the button on the headset marker.

Thanks

**Comment by Hettylool** [comment]
just do it at once.still when you change the basestation's height,do it again.
you needn't do it everytime.

**Comment by Djneo134** [comment]
Yes, but sometimes, when SteamVr or Riftcat crashs, then the headset position needs to be defined again. It'ld be great to have an offset option in the driver which indicates that the headset is on on desk or a table and not on the floor. Cause the cables are too shorts to go behind my desk and put on the floor at a normal distance from the basestation.

If we take the measure once, then we could apply this offset (in inches or centimeters) directly in the driver (like we do in the steamVR room setup).

We could have just a field or a cursor to indicate the offset to apply from floor distance, directly in the option menu (the one we have when we click on the green icon in the lower right side of the windows screen).

I'm sure it'ld be a great option for all the users.
